modules/media_duplicate_validation/src/Plugin/MediaDuplicateValidation/Md5.php

```diff
@@ -0,0 +1,103 @@
+<?php
+
+namespace Drupal\media_duplicate_validation\Plugin\MediaDuplicateValidation;
+
+use Drupal\file\Entity\File;
+use Drupal\media\Entity\Media;
+use Drupal\media\MediaInterface;
+use Drupal\media_duplicate_validation\Plugin\MediaDuplicateValidationBase;
+
+/**
+ * Duplication plugin to check md5 hash of the entire file contents.
+ *
+ * @MediaDuplicateValidation(
+ *   id = "md5"
+ * )
+ */
+class Md5 extends MediaDuplicateValidationBase {
+
+  const DATABASE_TABLE = 'media_validate_md5';
```

Comment:
This table name should be prefixed with the module name.

Reply:
:boom: